kubernetes/processbot/values-staging.yaml

```diff
@@ -1,5 +1,12 @@
 environment: staging
-ingressDomain: processbot.parity-stg.parity.io
+ingress:
+  domain: processbot.parity-stg.parity.io
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-http01
+    external-dns.alpha.kubernetes.io/target: traefik-external.parity-stg.parity.io.
```

Comment:
this annotation is not needed and confusing. the external dns annotation is used to **register** these dns records - which is already there deployed from the traefik itself.